src/Sulu/Component/Content/Compat/Structure/PageBridge.php

```diff
@@ -46,7 +46,10 @@ public function getLanguageCode()
 
     public function getCacheLifeTime()
     {
-        return $this->structure->getCacheLifetime();
+        /** @var array{type: string, value: string} $cacheLifetime */
+        $cacheLifetime = $this->structure->getCacheLifetime();
+
+        return $cacheLifetime;
```

Comment:
Would be better to type this correclty in the `StructureMetadata` class instead of have the need to set here `array{type: string, value: string}`: https://github.com/sulu/sulu/blob/668fd7cd8819407b546c1f376cae98deaf039077/src/Sulu/Component/Content/Metadata/StructureMetadata.php#L52-L63